docs/quickstart.md

```diff
@@ -1,246 +1,67 @@
-    "accountId": "123456789012"
-}
-```
-
-The API response for a successful lease destroy looks like this:
-
-```json
-{
-    "accountId": "519777115644",
-    "budgetAmount": 20,
-    "budgetCurrency": "USD",
-    "budgetNotificationEmails": [
-        "john.doe@example.com"
-    ],
-    "createdOn": 1572381585,
-    "expiresOn": 1572382800,
-    "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-    "lastModifiedOn": 1572442028,
-    "leaseStatus": "Inactive",
-    "leaseStatusModifiedOn": 1572442028,
-    "leaseStatusReason": "Destroyed",
-    "principalId": "jdoe123"
-}
-```
+Deploy DCE and lease an account quickly using the DCE CLI.
+
+1. Download the appropriate executable for your OS from the [latest release](https://github.com/Optum/dce-cli/releases/latest). e.g. for mac, you should download dce_darwin_amd64.zip
+
+1. Unzip and move the executable to a directory on your PATH, e.g.
+
+    ```
+    # Download the zip file
+    wget https://github.com/Optum/dce-cli/releases/download/<VERSION>/dce_darwin_amd64.zip
+
+    # Unzip to a directory on your path
+    unzip dce_darwin_amd64.zip -d /usr/local/bin
+    ```
+
+1. Type `dce init`. Leave all fields blank for now.
+
+1. Deploy DCE using IAM Credentials that have AdministratorAccess
+    ```
+    export AWS_ACCESS_KEY_ID=XXXXXXXXXX
+    export AWS_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXXXXXXX
+    dce system deploy
+    ```
+
+1. Retrieve the DCE API url from API Gateway in your master account, and add it to the dce config file, e.g.
+    ```
+    api:
+        host: abcdefghij.execute-api.us-east-1.amazonaws.com
+        basepath: /api
+    region: us-east-1
+    ```
+
+1. Prepare a second AWS account to be your first "DCE Child Account".
+    - Create an IAM role with `AdministratorAccess` and a trust relationship to your DCE Master Accounts
+    - Create an account alias by clicking the 'customize' link in the IAM dashbodard of the child account. This must not include the terms "prod" or "production".
+
+1. Add your child account to the accounts pool
+    ```
+    dce accounts add --account-id <child-account-id> --admin-role-arn <child-account-cross-account-role-arn>
+    ```
+
+1. Wait until the child account `accountStatus` is `Ready`
+    ```
+    dce accounts list
+    [
+        {
+            "accountStatus": "Ready",
+            "adminRoleArn": "arn:aws:iam::555555555555:role/DCEMasterAccess",
+            "createdOn": 1575485630,
+            "id": "775788068104",
+            "lastModifiedOn": 1575485630,
+            "principalPolicyHash": "\"bc5872b50475b186afea67ff47516a8f\"",
+            "principalRoleArn": "arn:aws:iam::775788768154:role/DCEPrincipal-quickstart"
+        }
+    ]
+    ```
+
+1. Lease your child account
+    ```
+    dce leases create --budget-amount 100.0 --budget-currency USD --email jane.doe@email.com --principal-id quickstartuser
+    ```
+
+1. Log in to your leased account
+    ```
+    dce leases login --open-browser <lease-id>
+    ```
```

Comment:
...."This will open a browser, and bring you to your AWS Account web dashboard"

... and maybe worth just an extra step, to show getting CLI creds (w/o `--open-browser` flag)